Source/EasyNetQ/IServiceProvider.cs

```diff
@@ -49,5 +49,7 @@ public interface IServiceRegister
         IServiceRegister Register<TService, TImplementation>()
             where TService : class
             where TImplementation : class, TService;
+
+        IServiceRegister UseDelayedMessageExchange();
```

Comment:
I see your point. Although what it does is to register an instance of `IScheduler` which supports delayed message exchanges. Perhaps the problem lies with method name. What would you say on `RegisterDelayedMessagesExchangeScheduler`?